go/git/interfaces.go

```diff
@@ -0,0 +1,11 @@
+package git
```

Comment:
@patrickxb this is really what I wanted to do, but everything else happened when I needed a `TeamIDWithVisibility`